sdk/metric/reader_test.go

```diff
@@ -35,6 +37,10 @@ type readerTestSuite struct {
 	Reader  Reader
 }
 
+func (ts *readerTestSuite) SetupSuite() {
+	otel.SetLogger(testr.New(ts.T()))
```

Comment:
Is it worth adding logr into the dependency tree when we only use it in tests?

Reply:
FWIW, we should have greater visibility into indirect dependencies when we update this branch to use Go 1.17 as the min Go version. It will structure go.mod files with indirect dependencies listed.